docs/CHANGELOG.md

```diff
@@ -6,12 +6,128 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## DevilutionX 1.2.0 (unreleased)
 ### Features
+- Hellfire support
+- Shareware support
+- Fully migrate saves between Diablo and Hellfire
+- Document [ini-file options](https://github.com/diasurgical/devilutionX/wiki)
+- Support file paths longer than 259 characters
+- Allow for up to 99 save games of each type
+- Much lower memory usage (now in line with the original)
+- Performance improvements
+- Gold picked goes to stack even when inventory is open
+- Pepin automatically heals
+- Reorganize ini-file
+- Show other players on the map
+- Widescreen menus
+- Scroll subtitles at the same speed as narration reader
+- Preserve list position when buy and selling items in stores
+- Use belt via the numpad
+- Close panels when pressing <kbd>ESC</kbd>
+- Shift-click to spend all stat points
+- Shift-click to clear readied spell
+- Shift-click consumables to move them between the belt and inventory
+- Shift-click equipment equip/unequip them
+- Preserve hotkeys and active spell across games
+- Set default active spell for new heroes
+- Apply sound volume to videos
+- DIABDAT.MPQ may now be upper case
 - Improved gamepad support:
   - D-Pad / Left stick navigation now works in all menus and is more responsive.
   - START + SELECT now opens the main menu.
   - Right stick mouse emulation has been improved.
   - Gamepads plugged in while the game is running can be used.
   - All connected gamepads can be used (previously, only the first one).
+- Add `--ttf-dir` and `--ttf-name` to allow for specifying a different UI font
+
+### Optional improvements
+- Improved transparency
+- Experience bar
+- Enemy health bar
+- Auto equip all item types
```

Comment:
Would suggest adding "on pickup" to avoid potential confusion
```suggestion
- Auto equip all item types on pickup
```

Reply:
But it also happens when buying them :D

Reply:
```suggestion
- Auto equip all item types on pickup or purchase
```

Reply:
> But it also happens when buying them :D

🤣 I implemented it and forgot about it.